tox.ini

```diff
@@ -498,7 +498,7 @@ commands =
     tornado_contrib: pytest {posargs} tests/contrib/tornado
     vertica_contrib: pytest {posargs} tests/contrib/vertica/
 # run subsets of the tests for particular library versions
-    ddtracerun: pytest {posargs} tests/commands/test_runner.py
+    ddtracerun: pytest {posargs} --no-cov tests/commands/test_runner.py
```

Comment:
This change was required due to a deadlock occurring somehow with the pytest-cov plugin and debug logging enabled in the tracer. Removing the `basicConfig` calls in the tracer resolves the deadlock. We'll have to do some digging to find out exactly what's going on.